src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -191,11 +191,14 @@ public void SendAsync_HttpTracingEnabled_Succeeds()
 
                             // Do a post to a remote server
                             byte[] expectedData = Enumerable.Range(0, 20000).Select(i => unchecked((byte)i)).ToArray();
-                            HttpContent content = new ByteArrayContent(expectedData);
-                            content.Headers.ContentMD5 = TestHelper.ComputeMD5Hash(expectedData);
-                            using (HttpResponseMessage response = await client.PostAsync(Configuration.Http.RemoteEchoServer, content))
+                            foreach (Uri remoteServer in new[] { Configuration.Http.RemoteEchoServer, Configuration.Http.SecureRemoteEchoServer })
```

Comment:
Might be better to parameterize the test with this.  We will get intermittent failures sooner or later, and if the test is parameterized by server name then it's easy to see what server failed and decide if that's important or not.

Reply:
Ok